backend/Controllers/LoginController.cs

```diff
@@ -34,13 +34,9 @@ IUserService userService
 
             try
             {
-                var encodedUsernamePassword = authHeader.Substring("Basic ".Length).Trim();
-                string usernamePassword = AuthHelper.Base64Decode(encodedUsernamePassword);
-                int separatorIndex = usernamePassword.IndexOf(':');
-
-                var splitUsernamePassword = usernamePassword.Split(':');
-                var username = splitUsernamePassword[0];
-                var password = splitUsernamePassword[1];
+                (string, string) usernamePassword = AuthHelper.GetUsernameAndPassword(authHeader);
+                string username = usernamePassword.Item1;
+                string password = usernamePassword.Item2;
```

Comment:
Nice use of tuples! To make it _really_ nice, though, you can actually [give names to the fields in a tuple](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples#use-cases-of-tuples) and also [use deconstruction](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/builtin-types/value-tuples#tuple-assignment-and-deconstruction) similar to in JavaScript:
```suggestion
                (string username, string password) = AuthHelper.GetUsernameAndPassword(authHeader);
```